client/src/app/site/common/components/start/start.component.html

```diff
@@ -1,13 +1,36 @@
-<os-head-bar>
+<os-head-bar
+    [hasMainButton]="canManage()"
+    [editMode]="isEditing"
+    [mainButtonIcon]="'edit'"
+    [isSaveButtonEnabled]="startForm.valid"
+    (mainEvent)="editStartPage()"
+    (saveEvent)="saveChanges()"
+    (cancelEditEvent)="isEditing = !isEditing"
+>
     <div class="title-slot">
         <h2 translate>Home</h2>
     </div>
 </os-head-bar>
 
-<mat-card class="os-card">
+<mat-card *ngIf="!isEditing" class="os-card">
```

Comment:
as before, try to only use mat-card once use another empty container for `*ngIfElse`